app/models/bookings/placement_request/cancellation.rb

```diff
@@ -86,24 +88,47 @@ def cancelled_by_candidate?
   end
 
   def rejection_description
-    humanised_rejection_category || reason
+    humanised_rejection_categories&.join(" ")
   end
 
-  def humanised_rejection_category
-    return nil if rejection_category == 'other' || rejection_category.nil?
-
-    I18n.t(
-      %w[
-        helpers
-        label
-        bookings_placement_request_cancellation
-        rejection_category_options
-      ].push(rejection_category).join('.')
-    )
+  def humanised_rejection_categories
+    SCHOOL_REJECTION_REASONS
+      .excluding(:other)
+      .select { |a| self[a] }
+      .map { |rejection_category|
+        I18n.t([
+          "helpers",
+          "label",
+          "bookings_placement_request_cancellation",
+          "#{rejection_category}_options",
+          1
+        ].join('.'))
+      }
+      .push(reason)
+      .reject(&:blank?)
```

Comment:
`reject(&:blank?)` can be replaced by `compact`

Reply:
This is for the `reason` attribute; we were checking for blank previously so I kept it consistent; we may be able to `compact` instead if its always `nil` but I wouldn't bet on it so I've left it as-is